code/game/machinery/computer/Operating.dm

```diff
@@ -30,6 +35,13 @@
 		S.use_power = FALSE
 		internal_bodyscanner = S
 
+/obj/machinery/computer/operating/Destroy()
+	if(table)
+		table.computer -= src
```

Comment:
You do the `-= src` thing if they're lists. Since they're just objects contained within a non-list you should do:

```suggestion
		QDEL_NULL(table.computer)
```

For each of them to remove the reference and delete the object.

Reply:
In this case, ``QDEL_NULL()`` is only for the ``Destroy()``, right? The previous ``qdel()`` stuff in the ``machinery_process()`` remains as is?

Reply:
You should use `QDEL_NULL()` there too I think.

Reply:
It's all been ``QDEL_NULL()``-ified!